### DISCUSSION TITLE
Can I rerender only when react props change, instead of render every 16ms?

### DISCUSSION THREAD

**Discussion by linonetwo** [michalochman/react-pixi-fiber#237] [Q&A] [answered]
I'm creating a Map Editor, which probably doesn't need to rerender every 16ms.

Is there a simple way to disable the game loop when there are no props changes? Or, is this game loop resource-consuming? If it is cheap, I think it is OK to just keep it.

I may create a custom ticker and pass it into `options.sharedTicker` to use it, and this ticker should be react props aware...

**Comment by linonetwo** [comment, marked as answer]
Oh, seems @iandanforth post this question as an issue before https://github.com/michalochman/react-pixi-fiber/issues/48

**Reply by michalochman** [reply]
@linonetwo does that answer your question?

**Reply by linonetwo** [reply]
Yes, I see his docs about this https://github.com/iandanforth/thought-experiment/tree/master/app#avoid-calling-render

And he is doing update here https://github.com/iandanforth/thought-experiment/blob/c7f8a56d372a6cc5bb1e65d1899e8f180b0c1052/app/src/features/home/NetWrapper.js#L48

**Reply by linonetwo** [reply]
And use `usePixiApp` to get app from react-pixi-fiber
